### DISCUSSION TITLE
本地启动异常

### DISCUSSION THREAD

**Discussion by springliyadong** [ling7334/hoppscotch-backend-go#8] [Q&A] [answered]
你好！我在本地拉取分支，启动后发现问题，hoppscotch-selfhost-web这个工程中，settings.api.ts文件中引用了api中的generated文件夹下面的js，但是该文件夹都没有，编译不通过。烦请帮忙核实，是我那里不正确吗。如图：
(https://github.com/user-attachments/assets/fa0cb184-f342-4179-9535-f0f64499412a)
![微信图片_20240914150827](https://github.com/user-attachments/assets/9c6601fb-f013-4f62-b30e-1766e67e1370)

**Comment by ling7334** [comment, marked as answer]
这是hoppscotch项目的吧，你要不去[原项目](https://github.com/hoppscotch/hoppscotch)问问。
看起来generated文件夹，应该是编译的时候程序生成的吧。
